frontend/src/container/AllError/index.tsx

```diff
@@ -186,7 +213,7 @@ function AllErrors(): JSX.Element {
 			title: 'Exception Type',
 			dataIndex: 'exceptionType',
 			key: 'exceptionType',
-			...getFilter(onExceptionTypeFilter, 'Search By Exception'),
+			...getFilter(onExceptionTypeFilter, 'Search By Exception', 'exceptionType'),
```

Comment:
do we need 3 parameter ?

Reply:
Yes the third parameter is for filtering